packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelog.js

```diff
@@ -91,12 +111,29 @@ module.exports = function generateChangelog( newVersion = null ) {
 
 				// Commit the changelog.
 				tools.shExec( `git add ${ utils.changelogFile }`, { verbosity: 'error' } );
-				tools.shExec( `git commit -m "Docs: Changelog. [skip ci]"`, { verbosity: 'error' } );
+
+				let commitMessage;
+
+				if ( options.isDevPackage ) {
+					commitMessage = `Changelog for "${ packageJson.name }".`;
```

Comment:
Why committing every changelog separately? That's exactly one releasing sub packages should work differently than sub repos – in that case, there should be just one commit. Taken the number of code branching in this function you may have a perfect proof that it just shouldn't be the same function.